test/e2e/logs/fluentbit/modify_timestamp_date_format_test.go

```diff
@@ -0,0 +1,91 @@
+//go:build e2e
+
+package fluentbit
+
+import (
+	"net/http"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	testutils "github.com/kyma-project/telemetry-manager/internal/utils/test"
+	"github.com/kyma-project/telemetry-manager/test/testkit/assert"
+	kitk8s "github.com/kyma-project/telemetry-manager/test/testkit/k8s"
+	kitkyma "github.com/kyma-project/telemetry-manager/test/testkit/kyma"
+	. "github.com/kyma-project/telemetry-manager/test/testkit/matchers/log"
+	"github.com/kyma-project/telemetry-manager/test/testkit/mocks/backend"
+	"github.com/kyma-project/telemetry-manager/test/testkit/mocks/loggen"
+	"github.com/kyma-project/telemetry-manager/test/testkit/periodic"
+	"github.com/kyma-project/telemetry-manager/test/testkit/suite"
+)
+
+var _ = Describe(suite.ID(), Label(suite.LabelLogsFluentBit), Ordered, func() {
+	var (
+		mockNs           = suite.ID()
+		pipelineName     = suite.ID()
+		backendName      = "backend"
+		backendExportURL string
+	)
+
+	makeResources := func() []client.Object {
+		var objs []client.Object
+		objs = append(objs, kitk8s.NewNamespace(mockNs).K8sObject())
+
+		logProducer := loggen.New(mockNs).WithUseJSON()
+		objs = append(objs, logProducer.K8sObject())
+
+		backend := backend.New(mockNs, backend.SignalTypeLogs, backend.WithName(backendName))
+		backendExportURL = backend.ExportURL(suite.ProxyClient)
+		objs = append(objs, backend.K8sObjects()...)
+		logPipeline := testutils.NewLogPipelineBuilder().
+			WithName(pipelineName).
+			WithIncludeContainers(loggen.DefaultContainerName).
+			WithIncludeNamespaces(mockNs).
+			WithHTTPOutput(testutils.HTTPHost(backend.Host()), testutils.HTTPPort(backend.Port())).
+			Build()
+		objs = append(objs, &logPipeline)
+
+		return objs
+	}
+
+	Context("When a logpipeline that modify date, and @timestamp field exist", Ordered, func() {
```

Comment:
How does it modify the date? Am not getting the "date modification" aspect in the test scenario. Is the app emmiting a log attribute "date"?

As far es I can see, you are only checking if the date attribute is emmitted, don't you? No modification is happening

Reply:
Maybe it is a bad name, it should be `modify @tmistamp and date field exist`, the test ensure the `date` field has iso8601 format and `@timestamp` field created.

Reply:
Maybe "Should ship logs with @timestamp and date field in iso8601 format to backend"? You still have a "modified" there which is confusing